gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -50,11 +50,6 @@
       <scope>provided</scope> <!-- to leave out of the all-deps jar -->
```

Comment:
That's copypasta from a core libraries pom.xml file. I'm not a Maven expert at all. You would know better than me whether or not it's necessary.